### DISCUSSION TITLE
Previous created ragapps breaks after pulling the latest version (multiple-ragapps)

### DISCUSSION THREAD

**Discussion by benjaminbascary** [ragapp/ragapp#205] [Q&A] [answered]
**Describe the bug**
When pulling the latest version of ragapp (added agents) it breaks previously created ragapps in the multiple-ragapps app because of missing agent config file

Newly generated ragapps are working properly. 

**To Reproduce**
1. Have a previous instance of multiple ragapps deployed.        
3. Pull the latest ragapp
4. Re build
5. See error when chatting or open the old ragapps settings

**Expected behavior**
Handle missing agent config file. Not give any agent options maybe? IDK

**Screenshots**
![image](https://github.com/user-attachments/assets/ae715904-82df-49a2-9837-57f05d70bbb8)
 
![image](https://github.com/user-attachments/assets/b7bdf945-e55f-4064-9d95-2c6655c43f6a)

**Comment by benjaminbascary** [comment]
In the second screenshot you can see the empty drawer

**Comment by leehuwuj** [comment, marked as answer]
@benjaminbascary Thanks for the detailed description! Could you let me know what kind of deployment you’re using? I’m assuming you deployed with a mounted folder, which might be overriding your current agent configuration (agents.yaml) with the new one. If that's the case, i think you can try copy the released default `agents.yaml` file here (https://github.com/ragapp/ragapp/blob/e7e789d372d062ce654cd12fa109c253699853bb/src/ragapp/config/agents.yaml) to your config folder.

**Comment by marcusschiesser** [comment]
Moving this to discussions as we won't provide an migration at the moment (slows down development)
